build/media-suite_compile.sh

```diff
@@ -1019,7 +1019,7 @@ if { [[ $aom = y ]] || [[ $libavif = y ]] || { [[ $ffmpeg != no ]] && enabled li
     do_uninstall include/aom "${_check[@]}"
     get_external_opts extracommands
     do_cmakeinstall video -DENABLE_{DOCS,TOOLS,TEST{S,DATA}}=off \
-        -DENABLE_NASM=on -DFORCE_HIGHBITDEPTH_DECODING=0 "${extracommands[@]}"
+        -DCONFIG_TUNE_VMAF=1 -DENABLE_NASM=on -DFORCE_HIGHBITDEPTH_DECODING=0 "${extracommands[@]}"
```

Comment:
you may want to make this a conditional, based on the same conditions that libvmaf is enabled (`[[ $ffmpeg != no ]] && enabled libvmaf `) since there's no guarantee that a person enabled aom will always have vmaf enabled

Reply:
I guess you are not able to get my point ....I will try again:

Well, AOM is one of the best AV1 encoders with many options/parameters to get the best results .... `--tune=` is one of them ...with total 7 possible options ...with `--tune=psnr` & `--tune=ssim` inbuilt .....others are:
`--tune=vmaf_without_preprocessing` `--tune=vmaf_with_preprocessing` ....there is also one of the best option i.e. `--tune=butteraugli`

You want to ask user an option to built AOM with vmaf in `media-autobuild_suite.bat` with it's corresponding code in `media-suite_compile.sh`? ...if that's the case then what about  `--tune=butteraugli`?
 ......**It's true that not everyone will use vmaf or butteraugli ...but I feel a comprehensive encoder like AOM should be built with all possible options** .....screenshot of my built AOM running with `--tune=vmaf_without_preprocessing`

![aomenc-vmaf](https://user-images.githubusercontent.com/55090624/175817227-5a264b82-1cbf-4266-841a-89360af2427a.PNG)

Reply:
> ....there is also one of the best option i.e. `--tune=butteraugli`

For tune vmaf there is av1an or ab-av1 - not only with aom, but a bunch of encoders.

However butteraugli has been lost, it would be nice to have a way to enable it because it's useful for avif (libavif) encoding, too. Currently it requires an older libjxl lib (0.8.2) - I had tried to git this 0.8.x branch but as a mediasuite newbie ran into numerous problems while trying to compile.

There are some tickets on libjxl and libavif - it would be great to enable butteraugili for aom even with current libjxl versions, even if if only for 8 bpp.